pentaho/src/test/java/pt/webdetails/cpf/MimeTypeHandlerTest.java

```diff
@@ -31,69 +29,27 @@
   private static final String EXTENSION_UNKNOWN_BY_ALL = "qwerty";
   private static final String EXPECTED_OUTCOME_UNKNOWN_BY_ALL = "application/unknown";
 
-  private Log logger = LogFactory.getLog( MimeTypeHandlerTest.class );
-
   @Test
   public void testGetExtensionOnlyKnownInPlatform() {
 
-    String actualOutcome = StringUtils.EMPTY;
-
-    try {
-
-      actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_PLATFORM );
+    String actualOutcome = MimeTypeHandler.getMimeTypeFromExtension( EXTENSION_ONLY_KNOW_IN_PLATFORM );
 
-      log( "testGetExtensionOnlyKnownInPlatform()", EXTENSION_ONLY_KNOW_IN_PLATFORM, EXPECTED_OUTCOME_IN_PLATFORM,
```

Comment:
Why did you remove this log?

Reply:
The JUnit infrastructure already provides the needed report when failing.
The use of try-catch-fail was due to earlier JUnit versions (and even then, it was not really necessary).